.env.sample

```diff
@@ -10,6 +10,8 @@ PORT_L2_EXECUTION_ENGINE_P2P=30306
 PORT_PROVER_SERVER=9876
 PORT_PROMETHEUS=9091
 PORT_GRAFANA=3001
+# Comma-separated profiles to be used by default (l2_execution_engine,proposer,prover)
+COMPOSE_PROFILES=l2_execution_engine,proposer
```

Comment:
Maybe default to just l2_execution_engine?

Reply:
good idea, thanks